### DISCUSSION TITLE
Will this work without RPI Zero?  (using pi 3)

### DISCUSSION THREAD

**Discussion by 3dPaulHarvey** [dz0ny/klipper-prusa-mk3s#12] [Q&A] [answered]
Hello, I am looking to install Klipper on my Prusa MK3S.  I dont have a pi Zero, i have  a Pi 3.  
I was wondering if the procedure would be the same?  Are there any other installation guides that would be more specific?  
Im browsing the "PrusaOwners" one  https://github.com/PrusaOwners/klipper/blob/master/docs/Installation.md

thanks!

**Comment by dz0ny** [comment, marked as answer]
Yes the procedure is the same.
